### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by robinMcA**

### CODE REVIEW

File: .github/workflows/on-push-update-test-env.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        git add --all
+        git config --global user.email "CI@agiledigital.com.au"
+        git config --global user.name "commit bot"
+        git commit -m "chore(ci) commit made by run $GITHUB_RUN_ID in workflow $GITHUB_WORKFLOW  in $GITHUB_REPOSITORY"
+        git config -l | grep 'http\..*\.extraheader' | cut -d= -f1 |  xargs -L1 git config --unset-all
+
+        git remote add ade-labs https://token:$GH_TOKEN@github.com/agiledigital-labs/argo-apps.git
+
+    - name: Push changes
+      run: git push ade-labs qa
```
Comment (SamMurphyDev) [inline .github/workflows/on-push-update-test-env.yml:49]: Is this always going to push for qa?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "semantic-release": "^17.2.1"
+  },
+  "release": {
+    "branches": [
+      "main",
+      {
+        "name": "qa",
+        "prerelease": true
+      },
+      "chore/ADE-489/add-cicd"
```
Comment (SamMurphyDev) [inline package.json:20]: ?

Comment (robinMcA) [reply package.json]: Fixed

File: .github/workflows/on-push-update-test-env.yml (hunk 1 above)
Comment (robinMcA) [reply .github/workflows/on-push-update-test-env.yml]: Yep, both dev and qa track that branch, The yq change should only affect one branch at a time.
Seems best to have a branch that tracks the state of the qa machine as a whole.

**Pull request closed without merging**